orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/backup.sls

```diff
@@ -23,7 +23,7 @@ add_root_role_to_database:
 
 backup_postgresql_db:
   cmd.run:
-    - name: /opt/salt/scripts/backup_db.sh {{salt['pillar.get']('disaster_recovery:object_storage_url')}} "" "" "" {{salt['pillar.get']('disaster_recovery:ranger_admin_group')}} "/var/lib/pgsql"
+    - name: /opt/salt/scripts/backup_db.sh {{salt['pillar.get']('disaster_recovery:object_storage_url')}} "" "" "" {{salt['pillar.get']('disaster_recovery:ranger_admin_group')}} {{salt['pillar.get']('disaster_recovery:database_name')}} "/var/lib/pgsql"
```

Comment:
I think this'll cause an issue when we don't provide the `database_name`. IIRC when that's not provided, the actual value here will be a space, and we'll have 6 arguments.

In that case, the 6th argument will be the log location base, and `DATABASENAME="${6-}"` in the backup script will evaluate to `/var/lib/pgsql`.

A possible way out of this might be to use named argument parameters rather than the positional ones, so that we wouldn't have to worry about ordering.